modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to test event streaming in the cloud: You can start streaming 30 seconds after clicking to create the cluster. 
+
+[NOTE]
+====
+To use our Beta release, Redpanda issues you enough credits to run Redpanda with reasonable throughput. These credits expire 14 days after you activate your tenant through email verification. For details, see <<Beta credits and limits>>. To ask questions, give feedback, or extend your trial, use the **#serverless** https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
+==== 
+
+== Prerequisites
+
+. https://redpanda.com/try-redpanda?section=cloud-trial[Sign up for a Beta trial^] of Redpanda Serverless.
+
+. Make sure you have the latest version of `rpk`. See xref:get-started:rpk-install.adoc[].
+
+== Create a Serverless cluster
```

Comment:
The gist of Serverless is fast onboarding (sign-in, initial setup, streaming data from your laptop, connecting to your app).

The fast cluster creation is just one of the steps.

Please change the content accordingly. We don't want to focus on the cluster creation only.